MapboxNavigation-SPM.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -60,17 +60,17 @@
         "repositoryURL": "https://github.com/mapbox/mapbox-maps-ios.git",
         "state": {
           "branch": null,
-          "revision": "ff273a812f0414537cfb6f55d4945bf377195269",
-          "version": "10.0.1"
+          "revision": "4c58ce320a716ef6b7337186076ef3d8279882f7",
```

Comment:
Should we also mention in a changelog that we now depend on Mapbox Maps SDK `v10.1.0`?

Reply:
I haven't changed the minimal version here. It's just resolved as it's already supported.
The question here is do we check that we don't start using anything from 10.1.0 later?

Reply:
Yep, I guess it makes sense. We also mark it as MapboxMaps `v10._x_` in our changelog, so it should be fine.

Reply:
And now I've realised that due to NavNative and Maps requirements for Common it's impossible to have Nav Native 80 and Maps 10.0.0 at the same time. So it makes no difference if we bump the minimal Maps SDK version here. And it might be even better for those who check dependencies manually by looking in the repo.